docs/ADL2/master07.11-adl_rulesNEW.adoc

```diff
@@ -41,16 +69,128 @@ The following sub-sections describe common categories of rule used in archetypes
 
 A common need is to assert an arithmetic relationship among various values located at different paths within an archetype. The following example specifies the summation of an Apgar score total from its parts, each defined as distinct nodes within the archetype. The `apgar_total_value`, `apgar_heartrate_value` etc are EL variables bound to the relevant paths within the main archetype definition.
 
+[tabs,sync-group-id=adl-example]
+====
+at-coded ADL2::
++
 [source, adl]
 ----
-archetype (adl_version=2.0.6; rm_release=1.0.3; generated; uid=f27fac48-3acb-4061-9619-c783fd8346ab)
+archetype (adl_version=2.4.0; rm_release=1.0.3; generated; uid=f27fac48-3acb-4061-9619-c783fd8346ab)
   openEHR-EHR-OBSERVATION.apgar.v1.0.1
 
 description
   lifecycle_state = <"unmanaged">
   original_author = <...>
   details = <...>
-    
+
+definition
+  OBSERVATION[at0000] ∈ {  -- Apgar score
+    data ∈ {
+      HISTORY[at0002] ∈ {
+        events ∈ {
+          POINT_EVENT[at0003] occurrences ∈ {0..1} ∈ { -- 2 minutes
+            offset ∈ {
+              DV_DURATION[at9000] ∈ {
+                value ∈ {PT1M}
+              }
+            }
+            data ∈ {
+              ITEM_LIST[at0001] ∈ {
+                items ∈ {
+                  ELEMENT[at0009] occurrences ∈ {0..1} ∈ {
+                    value ∈ {
+                      DV_ORDINAL[at9001] ∈ {
+                        [value, symbol] ∈ { -- apgar_respiratory_value
+                          [{0}, {[at0010]}],
+                          [{1}, {[at0011]}],
+                          [{2}, {[at0012]}]
+                        }
+                      }
+                    }
+                  }
+                  ELEMENT[at0005] occurrences ∈ {0..1} ∈ {...} -- apgar_heartrate_value
+                  ELEMENT[at0013] occurrences ∈ {0..1} ∈ {...} -- apgar_muscle_tone_value
+                  ELEMENT[at0017] occurrences ∈ {0..1} ∈ {...} -- apgar_reflex_value
+                  ELEMENT[at0021] occurrences ∈ {0..1} ∈ {...} -- apgar_skin_colour_value
+                  ELEMENT[at0025] occurrences ∈ {0..1} ∈ {
+                    value ∈ {
+                      DV_COUNT[at9002] ∈ {
+                        magnitude ∈ {|0..10|} -- apgar_total_value
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+          POINT_EVENT[at0026] occurrences ∈ {0..1} ∈ { -- 2 minutes
+            offset ∈ {
+              DV_DURATION[at9003] ∈ {
+                value ∈ {PT2M}
+              }
+            }
+            data ∈ {
+              use_node ITEM_LIST[at0049] /data[at0002]/events[at0003]/data[at0001]
+            }
+          }
+          POINT_EVENT[at0027] occurrences matches {0..1} matches {...}  -- 3 minutes
+          POINT_EVENT[at0028] occurrences matches {0..1} matches {...}  -- 5 minutes
+          POINT_EVENT[at0031] occurrences matches {0..1} matches {...}  -- 10 minuten
```

Comment:
"MInuten" should be capitalised as per German grammar. I'm kidding; change it to _minutes_ here as well as the original example. :)